doc/changes.md

```diff
@@ -1,5 +1,113 @@
 # Changelog
 
+
+## Release v0.8.0 (2022/12/28)
+
+[![Digital Object Identifier for PyGMT v0.8.0](https://zenodo.org/badge/DOI/10.5281/zenodo.7481934.svg)](https://doi.org/10.5281/zenodo.7481934)
+
+### Highlights
+
+* 🎉 **Eighth minor release of PyGMT** 🎉
+* Wrapped ternary
+* Easy access to various remote datasets
+* Add a intro tutorial for creating contour maps
+* Support tab auto-completion for all GMT defaults
```

Comment:
```suggestion
* Added support for tab auto-completion for all GMT default parameters
* Created functions to download GMT remote datasets
* Wrapped the ternary module
* Added an intro tutorial for creating contour maps
```

My take on how this should be ordered, since I think tab autocomplete is the biggest change, followed by the remote dataset functions. To keep it standardized, I think there should also be a verb of the same tense saying what happened (e.g. added, created, wrapped, etc.).

Reply:
Totally agree @willschlitzer these formulations definitely need some polishing 🙂. Did not work on this, as I was not sure, whether everbody is happy with these highlights.

Reply:
Would also be good to link to the PRs where these features were implemented, or the root issue in the case of the remote datasets (since there are many)!

Reply:
Thanks @weiji14. The links are added in commit https://github.com/GenericMappingTools/pygmt/pull/2272/commits/4573f3956745ff6f90e98957daa41c93de13839b.